### ISSUE TITLE
Ignore dotfiles

### ISSUE THREAD

**Issue by GeoffreyPlitt** [Aloshi/EmulationStation#318]
I use btsync to synchronize ROMS, and it creates dotfiles in my ROMS folders.

ES and the scraper sees these dotfiles as games, but they should be ignored.

Is there a way to ignore files starting with "." ?

**Comment by njt1982** [comment]
Did you find a way to skip these?

**Comment by njt1982** [comment]
You can run this over SSH/terminal in the roms folder... Removes all `.*` files...
```
find -type f -name '.*' -exec rm {} \;
```

**Comment by GeoffreyPlitt** [comment]
I don't want to delete them, BTSync needs them to keep syncing. I'd like the scraper to ignore them.

**Comment by njt1982** [comment]
Ahh I see - it would be nice to be able to exclude them.

**Comment by boottrax** [comment]
I agree,  The Sega MegaDrive games list is a mess.  Without being able to ignore dot files, it looks pretty bad.
